src/app.js

```diff
@@ -6,8 +6,91 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+// do i need you? -- I THINK SO
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
 
+// ALL THE FIELDS
+const TRIP_FIELDS = ['id', 'name', 'continent', 'about', 'category', 'weeks', 'cost'];
+
+const tripList = new TripList();
+let trip = new Trip();
+
+let tripTemplate;
+let tripDetails;
+
+// RENDERS
+const render = function render(tripList) {
+  const tripTbody = $('#trip-list');
+  tripTbody.html('');
+
+  tripList.forEach((trip) => {
+    const generatedHtml = tripTemplate(trip.attributes);
+    tripTbody.append(generatedHtml);
+  });
+};
+
+const renderDetails = function renderDetails(trip){
+  const detailsDiv = $('#trip-details');
+  detailsDiv.html('');
+  trip.fetch({
+    success: (model) => {
+      const generatedHTML = $(tripDetails(trip.attributes));
+      detailsDiv.append(generatedHTML);
+      console.log(trip);
+    },
+    error: (status, response) => {
+      const errors = ($.parseJSON(response.responseText))['errors'];
+    }
+  });
+  console.log(trip);
+};
+
+// JQUERY
 $(document).ready( () => {
-  $('main').html('<h1>Hello World!</h1>');
+  // my templates
+  tripTemplate = _.template($('#trip-template').html());
+  tripDetails = _.template($('#detail-template').html());
+
+  console.log(`About to fetch data from ${ tripList.url }`);
+
+  tripList.on('update', render);
```

Comment:
You also need an:
```
tripList.on('sort', render);
```

Otherwise it sorts, but doesn't display the results on the screen!